package.json

```diff
@@ -47,7 +48,8 @@
     "hooks:uninstall": "node node_modules/husky/husky.js uninstall",
     "hooks:install": "node node_modules/husky/husky.js install",
     "lint": "eslint .",
-    "prettier": "prettier --check 'src/**/*.js'",
+    "prettier:check": "prettier --check 'src/**/*.js'",
```

Comment:
Make sure that we are not using `prettier` without `:check` in any of the scripts or something.